### PR TITLE
Feat: new badges component

### LINKED ISSUE #48: feat: light rounded badge
- add new badge component
- utilize the new badge - to add new tag next to title of the component section, to differentiate newly added components 👍 
image

### PR DESCRIPTION
**Pull request by tomasvn**
- CLOSES #48

### CODE REVIEW

**Comment by Charlie85270** [comment]
Hi Ngoc Tu, 
thanks for your request, the "new" badges on new components it's a good idea ! 
I have juste update the lint rules with eslint and prettier, it has caused conflicts with your PR. 
As soon as you've solved them we can merge.

**Comment by tomasvn** [comment]
@Charlie85270 resolved merge

File: components/kit/components/elements/badges/Badge.tsx [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
-                    viewBox="0 0 1792 1792"
-                    xmlns="http://www.w3.org/2000/svg"
-                >
-                    <path d="M1343 12v264h-157q-86 0-116 36t-30 108v189h293l-39 296h-254v759h-306v-759h-255v-296h255v-218q0-186 104-288.5t277-102.5q147 0 228 12z" />
-                </svg>
-            )}{' '}
+          <path d="M1343 12v264h-157q-86 0-116 36t-30 108v189h293l-39 296h-254v759h-306v-759h-255v-296h255v-218q0-186 104-288.5t277-102.5q147 0 228 12z" />
+        </svg>
+      )}
+            {' '}
```
Comment (Charlie85270) [inline components/kit/components/elements/badges/Badge.tsx:36]: `{' '}` is useless here you can remove it

**Review by Charlie85270 — CHANGES_REQUESTED** [review]
Top ! 
image

1. Label seam to not be vertical centered
2. I added the formatting rules (eslint and prettier) at the root of the project, can you lint your files please ? (yarn lint command). 

Regarding 2, I will add a hook on commit to automatically lint files for next time ;)

**Comment by tomasvn** [comment]
also check out [lint-staged](https://github.com/okonet/lint-staged)

**Pull request merged**